.github/workflows/release-checks.yml

```diff
@@ -0,0 +1,142 @@
+name: Release checks
+
+on:
+  pull_request:
+    paths:
+      - .github/workflows/release-checks.yml
+  pull_request_target:
```

Comment:
Might be a good idea to keep this in sync with `eval.yml` -- that is, trigger on undrafts -- just so we can pick up and test any new merge commit that `eval.yml` is evaluating.

Reply:
Why didn’t we just do that in the first place, out of curiosity? I was under the impression that it just wasn’t possible since we didn’t do it, given it’s a pretty obvious optimization. ;)

That may be a Silvan question and not you, though.

Reply:
Also, wait, doesn’t that have the downside of not testing a later merge, leading CI to be green when it shouldn’t be? Maybe we *should* keep the current behavior for now, in lieu of a proper merge queue?